clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/coredns_config

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  Corefile: |
+      .:5353 {
+        auto{
+            IN NS bastion-ppc64le-libvirt
+
+            *.libvirt-ppc64le-0-0                 IN A 172.30.213.49
```

Comment:
Is it possible to use the service URLs instead of the internal IPs as the alias records?